### ISSUE TITLE
Masked 3D transposed convolutions

### ISSUE THREAD

**Issue by devashishlohani** [suflaj/masked-convolution#2]
Hi,
Thanks for this excellent work.
Can you tell me how can we implement masked [3D transposed convolutions](https://pytorch.org/docs/stable/generated/torch.nn.ConvTranspose3d.html) ?
Thanks a lot in advance.

**Comment by suflaj** [comment]
Duplicate of #1

**Comment by suflaj** [comment]
This module is not concerned with those, and judging by the fact that transposed convolutions are a fair bit different than ordinary convolutions another one should probably be created.

As this is in minimal maintenance mode I doubt I would personally implement it, but contribution is welcome if someone wishes to add this functionality.

You can resubmit an issue as a feature request if you want.

EDIT: Alternatively, if all you want is to mask just the kernel, then all that is needed is adding the transposed convolution class to the list of allowed classes, in which case I suggest you also submit a feature request and I'll add them, however note that this is not the reverse operation of a masked convolution.
